### DISCUSSION TITLE
AttributeError: open_esm_datastore

### DISCUSSION THREAD

**Discussion by anuragap98** [intake/intake-esm#329] [Q&A]
Hi,
I think there is some issue with attribute of intake-esm. I am trying extract LENS DATA but keep getting error -
AttributeError: open_esm_datastore

**Comment by andersy005** [comment]
Hi @anuragap98, 

How did you install `intake-esm` via `pip`? or `conda`?

**Reply by anuragap98** [reply]
while using Colab and install through `pip`. I didn't try in windows shell but I am guessing colab isn't supporting `intake-esm`.

**Reply by andersy005** [reply]
Any update on this issue, @anuragap98? Were you able to resolve it?
